build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id "java"
     id "pmd"
-	id "edu.wpi.first.GradleRIO" version "2023.1.1"
+	id "edu.wpi.first.GradleRIO" version "2023.2.1"
```

Comment:
please undo this change. we don't want to update the version, as it could screw up code in other branches